src/stylesheets/_navigation.scss

```diff
@@ -71,7 +73,7 @@
     }
 
     .slide-in-panel__content {
-        padding: 0 0 1.2rem 0;
```

Comment:
I think it's more clear if not writing a single value or double value to specify all 4 sides.

Reply:
If you feel very strongly about this, we could change the https://stylelint.io/user-guide/rules/shorthand-property-no-redundant-values rule, but I would prefer to keep it this way.